frontend/packages/shared/src/components/LabeledTextField/LabeledTextField.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  CustomTextField,
+  CustomTextFieldProps,
+} from '../CustomTextField/CustomTextField';
+import { Text } from '../Text/Text';
+
+export type LabeledTextFieldProps = {
+  id: string;
+  label: string;
+} & Omit<CustomTextFieldProps, 'label'>;
```

Comment:
En oo varma mut pitäiskö olla `Omit<CustomTextFieldProps, 'id' | 'label'>;` ?

Reply:
Ei sinänsä tarvitse, toisaalta ei kyllä tuota labeliakaan tarvitse ottaa pois kun tuossa vain rajoitetaan niiden tyyppejä. Voisin selkeyden nimissä ehkä ottaa tuon `Omit`in pois.